tests/node_eviction_test.go

```diff
@@ -0,0 +1,262 @@
+package tests_test
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"kubevirt.io/kubevirt-ansible/tests"
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+const (
+	cirrosImageDev  string = "registry:5000/kubevirt/cirros-registry-disk-demo:devel"
```

Comment:
id like to avoid specifying the images to run on in each test file
can we use one image for all (and when needed we can change in one place to test other OSs)?

Reply:
This is an issue to us, as "registry:5000/kubevirt/cirros-registry-disk-demo:devel" is not working for us. I'm refactoring it to move these two consts to util.go, so other tests can use it as well.